rubygems.js

```diff
@@ -28,3 +28,19 @@ export async function rubygemsUpdate(rubygemsVersionInput, rubyPrefix) {
 
   return true
 }
+
+async function rubygemsLatest(gem, platform, engine, version) {
+  if (engine === 'ruby') {
+    if (version >= '3.0') {
+      await exec.exec(gem, ['update', '--system'])
+    } else if (version >= '2.6') {
+      await exec.exec(gem, ['update', '--system', '3.4.22'])
+    } else if (version >= '2.3') {
+      await exec.exec(gem, ['update', '--system', '3.3.27'])
+    } else {
+      console.log(`Cannot update RubyGems for Ruby version ${version}`)
+    }
+  } else {
+    exec.exec(gem, ['update', '--system'])
```

Comment:
Missing `await`

Reply:
Sorry, that line is only for non MRI Ruby.  Given our/my lack of knowledge of JRuby, remove it and log a message that RubyGems is not updated on non MRI platforms?

Nevermind, I'll fix it as you've listed above.  So, same behavior as current with non MRI.